content/en/docs/architecture/ci-operator.md

```diff
@@ -624,6 +624,28 @@ Test owners are encouraged to migrate from the [`ipi-aws`](https://steps.ci.open
 the [hypershift-hostedcluster-workflow](https://steps.ci.openshift.org/workflow/hypershift-hostedcluster-workflow)
 which can be achieved simply by renaming the name of the workflow.
 
+### Allowing Tests to Access RedHat Intranet
+{{% alert title="LIMITED SUPPORT" color="warning" %}}
+`Accessing RedHat Intranet` is a pilot project, and the support from the Test Platform is limited.
```

Comment:
```suggestion
`Accessing Red Hat Intranet` is a pilot project, and the support from the Test Platform is limited.
```
Red Hat is always two words (https://www.redhat.com/en/about/brand/standards/naming-and-trademarks)